README.md

```diff
@@ -37,11 +37,11 @@ pinpoint| 1.8.0-RC1
 
     > Ubuntu 
 
-        sudo apt-get install automake bison flex g++ git libtool make pkg-config openssl libssl-dev 
+        sudo apt-get install automake bison flex g++ git libtool make pkg-config openssl libssl-dev python-dev libxml2-dev libxslt-dev
```

Comment:
I met this, "Note, selecting 'libxslt1-dev' instead of 'libxslt-dev'" on Ubuntu.